README.md

```diff
@@ -19,8 +19,10 @@ object-service.working-directory = "/tmp"
 # Publish container image to Google container registry
 * Set up auth for publishing docker image to GCR
 `gcloud auth configure-docker`
-* Publish welder
-`sbt server/docker:publish` (or `sbt server/docker:publishLocal` for local development)
+* Publish welder. 
+   * **IMPORTANT** make sure you have a new commit so that you don't override the HEAD hash in gcr
+   * For local development: `sbt server/docker:publishLocal`
+   * To prod or if you'd like to share the image:  `sbt server/docker:publish` 
```

Comment:
maybe remove `To prod`? We should never intend to push an image used for prod locally